controllers/novels.js

```diff
@@ -0,0 +1,46 @@
+const models = require('../models')
+
+const getAllNovels = async (request, response) => {
```

Comment:
This route throws an error

![image](https://user-images.githubusercontent.com/5775716/82481402-3eab9380-9aa3-11ea-9f90-97fc1d89e130.png)